packages/template-retail-react-app/app/static/translations/compiled/en-US.json

```diff
@@ -2337,12 +2337,36 @@
       "value": "Update"
     }
   ],
+  "product_view.label.aria.quantity_decrement": [
```

Comment:
Don't forget to bring these translation to en-GB.json file, or we will see missing translations in the console